test/test_fortran.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import numpy as np
+from aerobulk import flux_noskin
+
+@pytest.mark.parametrize('algo, expected', [
+    ('andreas', {
+        'ql':-74.4637756,
+        'qh':-14.4129944,
+        'evap':-2.62721038,
+        'taux':3.02770734E-02,
+        'tauy':0.0,
+        }
+    ),
+    ('ncar', {
+        'ql':-88.4781876,
+        'qh':-16.6969528,
+        'evap':-3.12166286,
+        'taux':3.58519591E-02,
+        'tauy':0.0,
+        }
+    ),
```

Comment:
These things will fail the code linter, but that should get fixed automatically...